negocio/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Categoria(models.Model):
+    nombre = models.CharField(max_length=200)
+
+    def __str__(self):
+        return self.nombre
+
+
+class Producto(models.Model):
+    id = models.IntegerField(primary_key=True)
+    categoria = models.ForeignKey(Categoria, on_delete=models.CASCADE)
+    nombre = models.CharField(max_length=200)
+    marca = models.CharField(max_length=100)
+    tamano = models.CharField(max_length=4)
+    destacado = models.BooleanField(default=False)
+    precio_unitario =  models.FloatField(default= 0.0)
+
+
+    def guardar(self):
+        self.save_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        return self.nombre
+
+class Venta(models.Model):
+    item = models.ForeignKey(Producto, on_delete = models.CASCADE)
+    cantidad = models.IntegerField(default = 0, null = True, blank = True)
+    precio = models.FloatField(default= 0.0)
```

Comment:
Acá creo que conviene usar decimal en vez de float, ya que decimal te permite establecer la cantidad de decimales, y creería (no lo probé), que va a mostrar algo así: 123.00, en lugar del 123.0 que muestra el float(que a mí me parece horrible, o 123 ó 123,00. Un solo decimal no)

Reply:
Totalmente de acuerdo con lo de decimal.
```suggestion
    precio = models.DecimalField(default= 0.0)
```